src/Applications/LDAS_App/process_hist.csh

```diff
@@ -54,6 +54,13 @@ endif
 if($NENS > 1) then
    set GridComp = ENSAVG
    sed -i 's|VEGDYN|'VEGDYN0000'|g' $HISTRC
+   sed -i 's|CATCHCNCLM45|'CATCHCNCLM450000'|g' $HISTRC
```

Comment:
I think I understand now why there's no matching entry for CATCHCNCLM40.  So that's fine.  But the `sed` replacement here should **not** be such that we end up pulling the variable CNFROOTC from ens member 0000.  This variable must come from ENSAVG for consistency with all the other outputs from `'GridComp'` (which eventually turns into ENSAVG, right?)
The somewhat awkward templating in `GEOSldas_HIST.rc` may be hitting a wall soon...